pysts/kerasts/blocks.py

```diff
@@ -185,6 +186,20 @@ def mlp_ptscorer(model, inputs, Ddim, N, l2reg, pfx='out', sum_mode='sum', extra
     return pfx+'mlp'
 
 
+def mulsum_ptscorer(model, inputs, Ddim, N, l2reg, pfx='out', sum_mode='sum'):
+    """ Element-wise features from the pair fed to a concat of sum and multiplication of inputs.
+    """
```

Comment:
Hmm, I still don't quite understand why we can't just special-case mlp_ptscorer for Ddim=0.   But we don't need to dwell on this, I can try to change that myself.